### DISCUSSION TITLE
How can I install this on a Debian based OS?

### DISCUSSION THREAD

**Discussion by grimsby1hull0** [pop-os/shell#1725] [Q&A]
I really want to install this on Debian but I'm unable to find a suitable install command on the github page, I'd really appreciate any help :)

**Comment by jacobgkau** [comment]
Try cloning the repo and running `make local-install`: https://github.com/pop-os/shell?tab=readme-ov-file#installation 

You may need to check out one of the other `master_` branches, depending on your GNOME Shell version, as detailed in that section of the README.

You will need to have typescript installed for that to work. It [looks like](https://packages.debian.org/search?searchon=sourcenames&keywords=typescript) Debian has it packaged as `node-typescript`.
